lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -1,70 +1,61 @@
-                    (dblclick)="copyToClipboard(property.displayValue)"
-                    matTooltipShowDelay="1000"
-                    [matTooltip]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
-                    [matTooltipDisabled]="isEditable"
-                    [attr.data-automation-id]="'card-textitem-value-' + property.key"
-                    (keydown)="undoText($event)">
-                <textarea matInput
-                    *ngIf="property.multiline"
-                    title="{{property.label | translate }}"
-                    [cdkTextareaAutosize]="true"
-                    [cdkAutosizeMaxRows]="1"
-                    [cdkAutosizeMaxRows]="5"
-                    class="adf-property-value"
-                    [placeholder]="property.default"
-                    [attr.aria-label]="property.label | translate"
-                    [formControl]="textInput"
-                    [attr.data-automation-id]="'card-textitem-value-' + property.key">
-                </textarea>
-
-                <button
-                    *ngIf="isEditable"
-                    matSuffix
-                    class="adf-textitem-clear-icon"
-                    [attr.aria-label]="'CORE.METADATA.ACTIONS.CLEAR' | translate"
-                    (click)="clearValue()">
-                    <mat-icon>cancel</mat-icon>
-                </button>
-                <button
-                    *ngIf="isEditable"
-                    matSuffix
-                    class="adf-textitem-edit-icon"
-                    [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate">
-                    <mat-icon>mode_edit</mat-icon>
-                </button>
 
+            <input matInput
+                *ngIf="!property.multiline"
+                class="adf-property-value"
+                [ngClass]="{
+                    'adf-property-value-editable': editable,
+                    'adf-property-value-has-error': isEditable && hasErrors
+                }"
+                title="{{property.label | translate }}"
+                [placeholder]="property.default"
+                [attr.aria-label]="property.label | translate"
+                [formControl]="textInput"
+                (dblclick)="copyToClipboard(property.displayValue)"
+                matTooltipShowDelay="1000"
+                [matTooltip]="'CORE.METADATA.ACTIONS.COPY_TO_CLIPBOARD' | translate"
+                [matTooltipDisabled]="isEditable"
+                [attr.data-automation-id]="'card-textitem-value-' + property.key"
+                (keydown)="undoText($event)">
+            <textarea matInput
+                *ngIf="property.multiline"
+                title="{{property.label | translate }}"
+                [cdkTextareaAutosize]="true"
+                [cdkAutosizeMaxRows]="1"
+                [cdkAutosizeMaxRows]="5"
+                class="adf-property-value"
+                [ngClass]="{'adf-property-value-editable': editable}"
```

Comment:
I don't see this class in scss file for this component. Is not that missing?

Reply:
https://github.com/Alfresco/alfresco-ng2-components/blob/b00d12406ff70a52c49a04151f3da7cd98b83c4d/lib/core/src/lib/card-view/components/card-view/card-view.component.scss#L63